vuepress/docs/next/docs/reference/entando-cli.md

```diff
@@ -173,6 +173,72 @@ You will need to provide an `env.json` file in the same directory where the bund
 
 See [this tutorial](../../tutorials/ecr/export-bundle-from-application.md) for more instructions on exporting a bundle including how to setup your `env.json`.
 
+## Profile Management
+ENT PROFILE is essentially a command to manage and switch between different configurations.
```

Comment:
Let's use the same style used elsewhere, just format this as ```ent profile``` rather using caps. We're talking scripts here so case matters.